charts/bandstand-test-runner/templates/_helpers.tpl

```diff
@@ -1,4 +1,8 @@
 {{- define "bandstand-test-runner.labels" -}}
+system-code: {{ default .Release.Name .Values.systemCode }}
+{{ if .Values.systemGroup }} system-group: {{  .Values.systemGroup }} {{ end }}
+git-repo: {{ .Values.gitRepo }}
```

Comment:
With this change (in all the helpers files) this would be backwards compatible, and wouldn't need a major version bump.
```suggestion
git-repo: {{ default .Release.Name .Values.gitRepo }}
```